pyspecdata/core.py

```diff
@@ -3306,10 +3321,7 @@ def set_error(self,*args):
         
         `set_error('axisname',error_for_axis)` or `set_error(error_for_data)`
 
-        `error_for_data` can be a scalar, in which case, **all** the data errors are set to `error_for_data`
-        .. todo::
-                several options below -- enumerate them in the documentation
-        '''
```

Comment:
I would leave the todo message (or, better, explain the ways it can be called)

Reply:
OK, sorry, I got confused, and thought you had already reverted the bad commit, but this was part of it